react/tic-tac-toe/src/redux/actions.js

```diff
@@ -0,0 +1,35 @@
+import TicTacToeService from "../services/TicTacToeService";
+import { createTypes, completeTypes, withPostFailure, withPostSuccess} from 'redux-recompose';
+
+
+export const actions = createTypes(completeTypes(['GET_MATCHES','LOG_IN']), '@TICTACTOE');
+
+const actionCreators = {
+  getMatches: () => {
+    return {
+      type: actions.GET_MATCHES,
+      service: TicTacToeService.getMatches,
+      target: 'matches',
+    };
+  },
+  logIn: (credentials) => {
+    return {
+      type: actions.LOG_IN,
+      service: TicTacToeService.logIn,
+      target: 'user',
+      payload: credentials,
+      injections: [
+        withPostSuccess((dispatch, response) => {
```

Comment:
I'd define the injections in a utils file

Reply:
:white_check_mark: